frontend/src/Components/Root.tsx

```diff
@@ -13,13 +14,34 @@ import { FadeReplace } from "./transitions/FadeReplace";
 /**
  * Props available on this component
  */
-export interface IRootProps extends RouteComponentProps, WithTranslation {
+export interface IRootProps {
 }
 
 /**
  * The component class
  */
-class Root extends React.PureComponent<IRootProps, {}> {
+class Root extends React.PureComponent<IRootProps & RouteComponentProps & WithTranslation, {}> {
+	public componentWillMount() {
```

Comment:
`componentWillMount` is deprecated and will be removed in react 17. Can we do this in `componentDidMount` instead?

Reply:
Thanks for reviewing, I've modified code use `componentDidMount ` instead.